TestCases/compliance-level-3/0058-feel-number-function/0058-feel-number-function.dmn

```diff
@@ -0,0 +1,194 @@
+    <decision name="decision004" id="_decision004">
+        <description>Tests FEEL expression: 'number("1,000,000.01", ":", "."))' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'number("1,000,000.01", ":", "."))'?</question>
+        <allowedAnswers>null (number)</allowedAnswers>
+        <variable typeRef="number" name="decision004"/>
+        <literalExpression>
+            <text>number("1,000,000.01", ":", ".")</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision004_a" id="_decision004_a">
+        <description>Tests FEEL expression: 'number("1,000,000.01", 123, "."))' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'number("1,000,000.01", 123, "."))'?</question>
+        <allowedAnswers>null (number)</allowedAnswers>
+        <variable typeRef="number" name="decision004_a"/>
+        <literalExpression>
+            <text>number("1,000,000.01", 123, ".")</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision004_b" id="_decision004_b">
+        <description>Tests FEEL expression: 'number("1,000,000.01", ",", ":"))' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'number("1,000,000.01", ",", ":"))'?</question>
+        <allowedAnswers>null (number)</allowedAnswers>
+        <variable typeRef="number" name="decision004_b"/>
+        <literalExpression>
+            <text>number("1,000,000.01", ",", ":")</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision004_c" id="_decision004_c">
+        <description>Tests FEEL expression: 'number("1,000,000.01", ",", 123))' and expects result: 'null (number)'</description>
+        <question>Result of FEEL expression 'number("1,000,000.01", ",", 123))'?</question>
+        <allowedAnswers>null (number)</allowedAnswers>
+        <variable typeRef="number" name="decision004_c"/>
+        <literalExpression>
+            <text>number("1,000,000.01", ",", 123)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision005" id="_decision005">
+        <description>Tests FEEL expression: 'number("1000000.01", null, "."))' and expects result: '1000000.01 (number)'</description>
+        <question>Result of FEEL expression 'number("1000000.01", null, "."))'?</question>
+        <allowedAnswers>1000000.01 (number)</allowedAnswers>
+        <variable typeRef="number" name="decision005"/>
+        <literalExpression>
+            <text>number("1000000.01", null, ".")</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision006" id="_decision006">
+        <description>Tests FEEL expression: 'number("1,000,000.01", null, "."))' and expects result: '1000000.00 (number)'</description>
+        <question>Result of FEEL expression 'number("1,000,000.01", null, "."))'?</question>
+        <allowedAnswers>1000000.00 (number)</allowedAnswers>
```

Comment:
minor typo, I believe is meant to be `.01` here

Reply:
As Homer would say. Doh!!

Reply:
no worries; these as mentioned are only minor typos, but personally I prefer to review stuff --also manually-- before merge ;)